### PR TITLE
Update questions

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

File: Teams/meeting-recording-expiration-feature.md [hunk 1]
```diff
@@ -95,3 +95,7 @@ This feature is solely meant as a lightweight housekeeping mechanism to reduce s
 **When will the file be deleted?**
 
 The file will be deleted within five days of the expiration date, though this isn't a strict guarantee.
+
+**The default expiration is 120 days but when I checked Teams Admin center it is set to 60 days, why??**
```
Comment (JohanFreelancer9) [inline Teams/meeting-recording-expiration-feature.md:99]:
```suggestion
**The default expiration is 120 days but when I checked Teams Admin center it is set to 60 days, why?**
```

File: Teams/meeting-recording-expiration-feature.md [hunk 2]
```diff
@@ -95,3 +95,7 @@ This feature is solely meant as a lightweight housekeeping mechanism to reduce s
...
+
+The way the Teams policies work is if any policy in the center is set by the admin, a cached snapshot is taken of all settings. If the admin set any attribute on the policy when we temporarily had a 60-day default, they will need to change it manually to 120 days. If they didn't have a custom setting before we set the 120, then they get the 120.
```
Comment (JohanFreelancer9) [inline Teams/meeting-recording-expiration-feature.md:101]:
```suggestion
The way the Teams policies work is if any policy in the center is set by the admin, a cached snapshot is taken of all settings. If the admin set any attribute on the policy when we temporarily had a 60-day default, they will need to change it manually to 120 days. If they didn't have a custom setting before we set the 120 days default, then they get the 120 days.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**